build.psake.ps1

```diff
@@ -243,7 +243,11 @@ Task Test -depends Build {
     Import-Module Pester
     try {
         Microsoft.PowerShell.Management\Push-Location -LiteralPath $TestRootDir
-        Invoke-Pester
+        $TestResult = Invoke-Pester -PassThru -Verbose:$VerbosePreference
```

Comment:
Output stream output is captured in the variable.  Are you saying Invoke-Pester also writes directly to the host?